### ISSUE TITLE
Evaluate PR process for the PMRoK wiki content

### ISSUE THREAD

**Issue by jdmedlock** [Chingu-cohorts/pmrok#13]
**_Issue Description & Expected Outcome:_** 
Before allowing update access to the PMRoK wiki by all PM's a review mechanism must be established to ensure that major changes have the approval of the group.

**_Symptoms:_**
Not doing this can result in "rip & replace" changes impacting large parts of the content which don't reflect best practices adopted by the PM team.

**_Steps to Recreate:_** 
N/a

**_Resolution:_** 
Review the following to see if it will allow the PR process in GitHub to be used for this purpose. http://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html

**Comment by jdmedlock** [comment]
A test has been set up using the following repos:
- https://github.com/jdmedlock/wikiprtest-wiki is a repo containing a wiki for testing
- https://github.com/jdmedlock/wikiprtest-collab is a repo cloned from the wiki and which will be used to make changes and process PR's

TravisCI integration has been added to push changes from the `master` branch in wikiprtest-collab  to the `master` branch in wikiprtest-wiki whena PR is merged. In addition, the `master` branch of wikiprtest-collab has been set up as a protected branch to require reviews _before_ PR merges are performed.

To perform a test of this process:
1. Contact @jdmedlock to be added as a collaborator on the wikiprtest-collab repo
2. Clone wikiprtest-collab on your PC.
3. Make changes to `Home.md` or `Another-Page.md`
4. Commit your changes and push to the `development` branch of the wikiprtest-collab repo
5. Create a PR in wikiprtest-collab and assign @jdmedlock as the reviewer
6. Once the PR is approved, merge it and then verify that the changes were pushed to the wiki in the wikiprtest-wiki repo.

**Comment by jdmedlock** [comment]
Note that no additional files, such as a `readme.md` should be added to the mirror repo since it will be copied to the wiki when changes are merged into the `master` branch.
